torch/csrc/api/include/torch/detail/TensorDataContainer.h

```diff
@@ -136,6 +137,7 @@ AT_FORALL_SCALAR_TYPES_AND3(Bool, Half, BFloat16, TENSOR)
       sizes_({(int64_t)values.size()}), \
       scalar_type_(at::k##S), \
       type_(TensorDataContainerType::Tensor) { \
+    at::tracer::impl::NoTracerDispatchMode tracer_guard; \
     at::AutoNonVariableTypeMode non_var_type_mode(true); \
```

Comment:
Delete AutoNonVariableTypeMode

Reply:
I'll revert changed to this file according to @yf225 's comment.

@yf225 - do you know why AutoNonVariableTypeMode is needed here?